webpack.config.client.js

```diff
@@ -1,6 +1,29 @@
 /* eslint-disable global-require */
 const AssetsPlugin = require('assets-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+const webpack = require('webpack');
+const dotenv = require('dotenv');
+
+const result = dotenv.config();
+const SIMORGH = /^SIMORGH_/i;
+
+if (result.error) {
+  throw result.error;
+}
+
+const getClientEnvVars = () => {
```

Comment:
Feedback on where this method could/should live so that it's easier to test would be useful